### PR TITLE
Added sort function to Chemmaster

### PR DESCRIPTION
**Pull request by DmitriTheDemon**
## About the PR
Sorted ChemMaster output window. I was asked to push this change upstream from a fork as others might desire it.

## Why / Balance
Quality of life change to allow finding reagents easier in ChemMaster

## Technical details
Added orderby to for each loop to sort contents

## Media
Space Wizard Sorted ChemMaster

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
N/A

**Changelog**

:cl:
- tweak: Sorted ChemMaster reagent list

### CODE REVIEW

**Comment by dvir001** [comment]
Before anyone say "But Frontier AGPL" he made the PR, he is making it MIT.

File: Content.Client/Chemistry/UI/ChemMasterWindow.xaml.cs [hunk 1]
```diff
@@ -182,7 +182,7 @@ private void UpdatePanelInfo(ChemMasterBoundUserInterfaceState state)
             };
             bufferHBox.AddChild(bufferVol);
 
-            foreach (var (reagent, quantity) in state.BufferReagents)
+            foreach (var (reagent, quantity) in state.BufferReagents.OrderBy(x => x.Reagent.Prototype))
```
Comment (0x6273) [inline Content.Client/Chemistry/UI/ChemMasterWindow.xaml.cs:185]: Shouldn't this sort by localized name?

**Comment by 7-62x54r** [comment]
Maybe make it an _option_ and not forced? I like to organize master by hand, it's not that hard...

**Comment by 0tito** [comment]
Id rather have an option to manually move the chems, making them automatically sorted will only make it more annoying

**Comment by lexaSvarshik** [comment]
Make it an option, because sometimes it's easier to use it as is, without sorting.
So. it will be like "Press that button to see chem sorted", "Press another button and return to original order"

**Comment by DmitriTheDemon** [comment]
It would seem this change is not desired upstream feel free to implement at a later point if that opinion changes.

**Pull request closed without merging**